### ISSUE TITLE
Icons in header documentation

### ISSUE THREAD

**Issue by colinreedmiller** [vmware/clarity#3616]
Documentation specifies 'not recommended' for icon only in header then displays an icon only header in the examples.

https://clarity.design/documentation/header

Recommendation:
![Screen Shot 2019-07-11 at 1.00.33 PM.png](https://images.zenhubusercontent.com/5bc78fe0ba9b001064ad7062/c7915448-1565-41ad-ba9f-de427353855d)

Header examples:
![Screen Shot 2019-07-11 at 1.00.45 PM.png](https://images.zenhubusercontent.com/5bc78fe0ba9b001064ad7062/4fa2eb5e-45d7-4f16-a32c-204a39c73097)

**Comment by andreiconstantinbytex** [comment]
Hello. I would like to be assigned to this issue. I will need further clarification though: is it enough to pull out of the webpage the icon-only example?

**Comment by Jinnie** [comment]
Hi Andrei,
You should remove both the icon-only image example, and the corresponding section of the sample code below the header images.

Thanks for helping with this!

**Comment by andreiconstantinbytex** [comment]
Should it also be taken out of the old documentation?

**Comment by Jinnie** [comment]
Thanks for so quickly submitting the change.
As you also updated the dev-app page, now Set 3 of the visual tests is failing.
Please check if the failing tests are relevant now and change them if needed or update the reference images.
Write back if you need any help with this.

**Comment by andreiconstantinbytex** [comment]
![image](https://user-images.githubusercontent.com/52167120/64168355-005a3f80-ce54-11e9-9f9c-b58ed8fc6681.png)

Since I removed the example in the middle with the logo icons this is why the test is failing. I would like to know what is the best way to edit these images so that the tests pass?

**Comment by hippee-lee** [comment]
Hi @andreiconstantinbytex - it looks like your changes affect the reference images for our visual regression tests in set 3. 

```
✘ navigation header-types default [chrome]
✘ navigation header-types header-dropdown [chrome]
```

You can update the reference images by running the command:
```
npm run fix:visual:light set3 && npm run fix:visual:dark set3
```

NOTE: you will need to have Docker installed and running on your system in order to run the gemini regression tests. You can [read more here](https://github.com/vmware/clarity/wiki/CSS-Regression-Testing) or let me know if you have a specific question.
